apps/iglidur-designer/src/app/libs/header/src/lib/header/header.component.scss

```diff
@@ -1,73 +1,78 @@
 .header {
-    height: 3.5rem;
-    background-color: rgb(54, 54, 54);
+  height: 3.5rem;
+  background-color: rgb(54, 54, 54);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 2rem;
+  .title {
     display: flex;
     align-items: center;
-    justify-content: space-between;
-    padding: 0 2rem;
-    .title {
+    justify-content: center;
+    gap: 1rem;
+    color: white;
+    .logo {
+      font-weight: 300;
+      font-size: 3rem;
+      text-decoration: line-through;
+      color: #f07d00;
+      margin: 0;
+    }
+    .title-text {
+      display: flex;
+      flex-direction: column;
+      font-size: 1rem;
+      .title-text-content {
+        margin: 0;
+      }
+      .lower {
+        color: rgb(179, 179, 179);
+      }
+    }
+  }
+  .menu-section {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 100%;
+    .menu-options {
+      display: flex;
+      height: inherit;
+      .menu-option {
+        color: white;
         display: flex;
         align-items: center;
-        justify-content: center;
-        gap: 1rem;
-        color: white;
-        .logo{
-            font-weight: 300;
-            font-size: 3rem;
-            text-decoration: line-through;
-            color: #f07d00;
-            margin: 0;
-        }
-        .title-text {
-            display: flex;
-            flex-direction: column;
-            font-size: 1rem;
-            .title-text-content {
-                margin: 0;
-            }
-            .lower {
-                color: rgb(179, 179, 179);
-            }
+        gap: 10px;
+        height: inherit;
+        cursor: pointer;
+        padding: 0 10px;
+        &:hover {
+          background-color: rgb(194, 188, 188, 0.2);
```

Comment:
It will also be nice if you could have your application colours as variables

Reply:
Refactored the code to use color variables